src/rgw/rgw_rados.cc

```diff
@@ -7408,7 +7413,7 @@ int RGWRados::apply_olh_log(RGWObjectCtx& obj_ctx, RGWObjState& state, const RGW
     ObjectWriteOperation rm_op;
 
     rm_op.cmpxattr(RGW_ATTR_OLH_ID_TAG, CEPH_OSD_CMPXATTR_OP_EQ, olh_tag);
-    rm_op.cmpxattr(RGW_ATTR_OLH_VER, CEPH_OSD_CMPXATTR_OP_GT, last_ver);
+    rm_op.cmpxattr(RGW_ATTR_OLH_VER, CEPH_OSD_CMPXATTR_OP_GT, last_ver + 1);
```

Comment:
makes sense, since we set it to last_ver earlier in the function. i just think it would be a little more clear if we used CEPH_OSD_CMPXATTR_OP_EQ to check that `last_ver == OLH_VER` instead of `last_ver+1 > OLH_VER`. what do you think?

Reply:
looks better, EQ makes more clear. I have commited again, please review